package.json

```diff
@@ -7,6 +7,7 @@
     "firebase": "^9.6.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-hot-toast": "^2.2.0",
```

Comment:
Thank you for introducing me to hot toast component! It's so satisfying seeing notifications pop-up and fade, and makes for clean code too.